src/styles.scss

```diff
@@ -1 +1,9 @@
-/* You can add global styles to this file, and also import other style files */
+/* You can add global styles to this file, and also import other style files */
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+*, *:before, *:after {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}

```

Comment:
Why are we doing a css reset?

Reply:
I use reset in my projects to achieve a consistent look and not be browser-dependent. I was taught this at bootcamp :D but I may not need it :D